### ISSUE TITLE
Dark mode broken really bad

### ISSUE THREAD

**Issue by lenovouser** [Studio3T/robomongo#1761]
OS: macOS Catalina 10.15.6 (19G73)
Version: Robo 3T 1.4.0

![2020_09_04_14_27_58@2x](https://user-images.githubusercontent.com/7549136/92239277-ff227f00-eeba-11ea-9568-e4bd1a007000.jpg)

**Comment by bhuvan5** [comment]
You are correct, the dark mode is horrible. Now I have to switch back to light mode in my MAC OS just for robo3T. Is this going to be fixed soon. This is a terrible customer experience.

**Comment by jamesloper** [comment]
Ahhh shoot I installed the new version over 1.3. Anyone have a link to download 1.3 again???

**Comment by stephen37** [comment]
> Ahhh shoot I installed the new version over 1.3. Anyone have a link to download 1.3 again???

@jamesloper You can find it in the release in the asset of the release of 1.3.1.
https://github.com/Studio3T/robomongo/releases/tag/v1.3.1

**Comment by crossinghoods** [comment]
+1 Here

**Comment by patrickvdvelden** [comment]
OS: macOS Catalina 10.15.6 (19G73)
Version: Robo 3T 1.4.0

Confirmed. Dark mode seems to be missing some dark background colors.

**Comment by anurag-rai** [comment]
UI seems ok for mac light mode. However, things are barely readable in dark mode.

**Comment by mr-kapoor** [comment]
Same

**Comment by maltahs** [comment]
+1

**Comment by watnotte** [comment]
+1

**Comment by abhishek-flexi** [comment]
+1

**Comment by danieler1981** [comment]
+1

**Comment by andreisaikouski** [comment]
+1 here.
Also the tree on the left is really hard to read
![image](https://user-images.githubusercontent.com/58610870/92268705-e90fc100-eeb0-11ea-8d34-24be57f08213.png)

**Comment by simsekgokhan** [comment]
Hi all, we are very sorry for this problem. It is probably the result of Qt upgrade in this version which was unplanned and we had to do it last minute. We will make sure that dark mode is checked carefully during dev. and QA in future versions.

This seems to fix it for us, can you try this:
```
// Disable dark mode for Robo 3T
defaults write com.3tsoftwarelabs.robo3t NSRequiresAquaSystemAppearance -bool yes
```

And as stated above, older releases are here:  
https://github.com/Studio3T/robomongo/releases/tag/v1.3.1

Also to reset Robo 3T theme (enable dark mode)
```
defaults delete com.3tsoftwarelabs.robo3t NSRequiresAquaSystemAppearance
 ```

**Comment by kakadais** [comment]
+1

**Comment by yeukfei02** [comment]
I have the same issue here, just update robo 3t. 
I am using in mac dark mode.

where to add this command?
`defaults delete com.3tsoftwarelabs.robo3t NSRequiresAquaSystemAppearance`

thanks.

**Comment by yeukfei02** [comment]
I type `defaults delete com.3tsoftwarelabs.robo3t NSRequiresAquaSystemAppearance` in my terminal.

```
2020-09-06 17:09:41.713 defaults[57036:443977]
Domain (com.3tsoftwarelabs.robo3t) not found.
Defaults have not been changed.
```

**Comment by yeukfei02** [comment]
now I download back 1.3 version for now..
wait for the updates.
thanks.

**Comment by xf8** [comment]
> > Ahhh shoot I installed the new version over 1.3. Anyone have a link to download 1.3 again???
> 
> @jamesloper You can find it in the release in the asset of the release of 1.3.1.
> https://github.com/Studio3T/robomongo/releases/tag/v1.3.1

I also encountered this problem, this link seems unable to download the 1.3 version

**Comment by yeukfei02** [comment]
@xf8 
here?
https://github.com/Studio3T/robomongo/releases/tag/v1.3.1

**Comment by xf8** [comment]
> @xf8
> here?
> https://github.com/Studio3T/robomongo/releases/tag/v1.3.1

Yes, thank you~

**Comment by FrostyLeaf** [comment]
The same. I'm already blind now.

**Comment by davidinon** [comment]
same :)

<img width="1440" alt="Screen Shot 2020-09-07 at 15 17 27" src="https://user-images.githubusercontent.com/5325091/92387185-c6ceab00-f11d-11ea-8cfd-1903bca1a3fd.png">
<img width="336" alt="Screen Shot 2020-09-07 at 15 18 48" src="https://user-images.githubusercontent.com/5325091/92387193-ca623200-f11d-11ea-8b85-b3d9fae76d63.png">

**Comment by gilles-yvetot** [comment]
@yeukfei02 
This is the command you need to paste:
`defaults write com.3tsoftwarelabs.robo3t NSRequiresAquaSystemAppearance -bool yes`

The one you mentioned is the one to re-enable the dark mode

**Comment by simsekgokhan** [comment]
Hi all, seems like Qt does have some problems with dark mode support. So, we are now preparing a patch version 1.4.1 to force the light mode on macOS (and to fix the broken macOS native theme/look #1766 as well). It is still in pre-release state, but it will be great if you can make a quick test and let us know if it works.

Steps to test the (pre-release) patch:  
- If you already disabled the dark mode using the work-around here https://github.com/Studio3T/robomongo/issues/1761#issuecomment-687281760, then first reset Robo 3T theme (to enable dark mode) 
`defaults delete com.3tsoftwarelabs.robo3t NSRequiresAquaSystemAppearance`
- Open Robo 1.4.0 and see the problematic dark mode 
- Download the (pre-release) patch version -> [Robo 1.4.1](https://github.com/Studio3T/robomongo/releases/tag/v1.4.1)
- Open Robo 1.4.1 - hopefully it will work and start in light mode (and with native look #1766).

Please let us know the result. Thanks in advance.

**Comment by lenovouser** [comment]
@simsekgokhan that fixed it, yeah 👍🏻

![2020_09_10_12_11_14@2x](https://user-images.githubusercontent.com/7549136/92716508-32538b00-f35f-11ea-9c4f-6b7b1ad62c94.jpg)

**Comment by patrickvdvelden** [comment]
Installed 1.4 on OS X 10.15.6 in Dark Mode: problems were visible.
Installed 1.4.1: GUI was automagically switched to light mode.

Works as intended!

**Comment by osadchiynikita** [comment]
@simsekgokhan Thanks team for the fix. On OS X 10.15.6 (19G2021) with Dark mode works as expected (light mode without bugs)

**Comment by oxr463** [comment]
+1

**Comment by ckdot** [comment]
+1

**Comment by marlonbrgomes** [comment]
+1

**Comment by kbellioum** [comment]
+1

**Comment by osystst** [comment]
please, fix it.

**Comment by AnatoleLucet** [comment]
On Linux (Manjaro), I had to switch my Qt theme to a light theme. To do so, open the Qt5 settings (can be opened via a terminal with `qt5ct`), and in the appearance tab, change the Color Scheme settings to something light. Then apply, and restart Robo 3T
